packages/flutter/test/widgets/media_query_test.dart

```diff
@@ -47,6 +47,7 @@ void main() {
     expect(data.invertColors, false);
     expect(data.disableAnimations, false);
     expect(data.boldText, false);
+    expect(data.platformBrightness, ui.Brightness.light);
```

Comment:
As before: I think Brightness should be available to all Flutter libraries without qualification.

Reply:
I changed this file's import to bring in Brightness without a qualifier.